jump-core/src/main/java/com/bitdecay/jump/collision/SATResolution.java

```diff
@@ -72,7 +72,7 @@ public void compute(BitBody body, BitBody otherBody, BitRectangle resolvedPositi
                                 float resolutionPosition = body.aabb.xy.plus(axisOver.axis.times(axisOver.distance)).dot(axisOver.axis);
                                 float lastPosition = body.lastPosition.dot(axisOver.axis);
 
-                                if (!MathUtils.sameSign(resolutionPosition, lastPosition) || Math.abs(lastPosition) < Math.abs(resolutionPosition)) {
+                                if (!MathUtils.sameSign(resolutionPosition, lastPosition) || (lastPosition < resolutionPosition)) {
```

Comment:
I over-complicated this by doing the math.abs() call. Since we get these values from dot products, they are already in the exact units we need for a direct comparison.